src/v0/consignment_notes.ts

```diff
@@ -0,0 +1,65 @@
+import { Client } from '../client'
+import { BaseError } from '../errors'
+import { UriHelper } from '../uri-helper'
+import { ThBaseHandler } from '../base'
+
+export interface ConsignmentNotesOptions {
+  user?: string
+  base?: string
+}
+
+export interface ConsignmentNotesPdfResponse {
+  data?: string
+  contentType?: string
+  filename?: string
+}
+
+export class ConsignmentNotes extends ThBaseHandler {
+  public static baseEndpoint = '/api/v0/consignment-notes'
+  endpoint: string
+  http: Client
+  public options: ConsignmentNotesOptions
+  public uriHelper: UriHelper
+
+  constructor (options: ConsignmentNotesOptions, http: Client) {
+    super(http, {
+      endpoint: ConsignmentNotes.baseEndpoint,
+      base: options.base ?? 'https://api.tillhub.com'
+    })
+    this.options = options
+    this.http = http
+
+    this.endpoint = ConsignmentNotes.baseEndpoint
+    this.options.base = this.options.base ?? 'https://api.tillhub.com'
+    this.uriHelper = new UriHelper(this.endpoint, this.options)
+  }
+
```

Comment:
Maybe you also would like to add `getAll` method? Because you will need it for another story

Reply:
Ah, good to know, wasn't aware of it. I'd still like to separate it in two different PRs since they're related to two different tasks (and different task ids).